src/components/inputs/ComponentAttributeInput.vue

```diff
@@ -1,23 +1,26 @@
 <template>
-  <q-toggle
-      v-if="inputType === 'boolean'"
+  <component
+      v-if="localAttribute"
+      :is="inputList[localAttribute.type]"
       v-model="localAttribute.value"
-      :label="$t(localAttribute.name)">
-  </q-toggle>
-  <q-input v-else
-           :type="inputType"
-           autogrow
-           v-model="localAttribute.value"
-           :label="$t(localAttribute.name)"></q-input>
+      :label="localAttribute.name"
+      :type="localAttribute.type">
+  </component>
 </template>
 
 <script setup>
 import {
-  defineProps, ref, defineEmits, watch,
+  ref, defineEmits, watch, defineAsyncComponent,
 } from 'vue';
 
 const emit = defineEmits(['update:attribute']);
 
+const inputList = {
+  Boolean: defineAsyncComponent(() => import('./BooleanToggle.vue')),
+  String: defineAsyncComponent(() => import('./TextMultiline.vue')),
+  Number: defineAsyncComponent(() => import('./TextMultiline.vue')),
```

Comment:
Why using text-multiline instead of input ?

Reply:
added aditional number input component